packages/library/src/commonMain/kotlin/io/realm/internal/SuspendableNotifier.kt

```diff
@@ -49,9 +46,9 @@ internal class SuspendableNotifier(private val owner: Realm, private val dispatc
     private val _realmChanged = MutableSharedFlow<RealmReference>(onBufferOverflow = BufferOverflow.SUSPEND, extraBufferCapacity = 1)
 
     // Must only be accessed from the dispatchers thread
-    private val realm: BaseRealm by lazy {
+    private val realm: BaseRealmImpl by lazy {
```

Comment:
Same. Is there a reason we are restricting these to the implementation class? or just a side-effect of refactoring in Android Studio?